com.gluonhq.SceneBuilder.yml

```
@@ -0,0 +1,58 @@
+app-id: com.gluonhq.SceneBuilder
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder-runner
+rename-desktop-file: scenebuilder-SceneBuilder.desktop
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --persist=.scenebuilder
+  - --filesystem=home
+cleanup:
+  - /share/info/*
+  - /share/locale/*
+  - /share/man/*
+  - /share/runtime/locale/*
+modules:
+  - name: SceneBuilder
+    buildsystem: simple
+    build-commands:
+    - mkdir -p deb-package
+    - ar p SceneBuilder.deb data.tar.xz | tar -xJf - -C deb-package
+    - mv deb-package/opt/scenebuilder /app
+    - install -Dm755 SceneBuilder-runner /app/bin/SceneBuilder-runner
+    - install -Dm644 com.gluonhq.SceneBuilder.metainfo.xml -t /app/share/metainfo/
+    - install -Dm644 com.gluonhq.SceneBuilder.256.png /app/share/icons/hicolor/256x256/apps/com.gluonhq.SceneBuilder.png
+    - install -Dm644 com.gluonhq.SceneBuilder.128.png /app/share/icons/hicolor/128x128/apps/com.gluonhq.SceneBuilder.png
+    - desktop-file-edit --remove-key=Categories /app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop
+    - desktop-file-edit --set-key=Categories --set-value='Development' /app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop
+    - desktop-file-edit --set-icon=com.gluonhq.scenebuilder /app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop
```

icon name are case sensitive like file names.
```suggestion
    - desktop-file-edit --set-icon=com.gluonhq.SceneBuilder /app/scenebuilder/lib/scenebuilder-SceneBuilder.desktop
```

---

already updated it